README.md

```diff
@@ -24,18 +24,20 @@ PyPRECIS is the python based training environment for Met Office PRECIS training
 
 
 ## Overview
-PyPRECIS forms part of the Met Office PRECIS regional climate modelling system.  It is principally designed as a learning tool to faciliate processing of regional climate model (RCM) output.  It is desgined to be used in conjunction with taught workshops in an instructor led environment.
+PyPRECIS is principally designed as a learning tool to faciliate processing of regional climate model (RCM) output.  It is desgined to be used in conjunction with taught workshops in an instructor led environment. 
 
 PyPRECIS is built on [Jupyter Notebooks](https://jupyter.org/), with data processing performed in python, making use of [Iris](https://scitools.org.uk/iris/docs/latest/).
 
-Further information about PRECIS can be found on the [Met Office website](https://www.metoffice.gov.uk/precis).
+The data analysed in the notebooks is from the CORDEX-Core simulations which provide an ensemble of high-resolution (at least 25 km) regional climate change information. Further information about CORDEX-Core can be found on the [CORDEX website](https://cordex.org/experiment-guidelines/cordex-core/cordex-core-simulations/).
 
 ## Contents
 The teaching elements of PyPRECIS are contained in the `notebooks` directory. The core primary worksheets are:
 
 Worksheet | Aims
 :----: | -----------
-[1](notebooks/worksheet1.ipynb) | <li>Identify and list the names of PRECIS output data in PP format using standard Linux commands</li><li>Use basic Iris commands to load data files, and view Iris cubes</li><li>Use Iris commands to remove the model rim, select data variables and save the output as NetCDF files</li>
+[1](notebooks/worksheet1.ipynb) | <li>Identify and list the names of CORDEX output data in netCDF format using standard Linux commands.
```

Comment:
Table formatting is broken here.  I think you need to avoid new lines for text that sits inside a cell

Reply:
fixed